test/parity-spec.js

```diff
@@ -0,0 +1,55 @@
+import { describeWithDOM } from './_helpers';
```

Comment:
what do you guys think about a test like this? this effectively tests that the API of methods on `shallow` is the same as the API of the same methods on `mount`, which seems like an important thing to test explicitly